### PR TITLE
Documentation guidelines update on the formatting of lists

### PR DESCRIPTION
**Pull request by havatv**
First try on how to format lists (start items with capital letter, period as punctuation - only for complex / compound items).  I have also tried to apply the "rules" to this document.
Input appreciated.
See also discussion in #3112.

### CODE REVIEW

File: source/docs/documentation_guidelines/writing.rst [hunk 1]
```diff
@@ -63,6 +63,17 @@ section, subsection and minisec.
...
 
+Lists
+-----
+
+Start all list items with a capital letter.
+
+Do not use punctuation after list items that only contain a single simple
+sentence.
+
+Use period (`.`) as punctuation for list items that consist of several
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/writing.rst]: double **`**

File: source/docs/documentation_guidelines/writing.rst [hunk 2]
```diff
@@ -63,6 +63,17 @@ section, subsection and minisec.
...
+sentences or is composed.
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/writing.rst]: "is composed"?
Sorry, I don't know if these (single simple, composed) are English grammatical naming but I think examples might help non English native writers.

Maybe, adding these rules as a list...?

File: source/docs/documentation_guidelines/writing.rst [hunk 3]
```diff
@@ -480,25 +491,25 @@ Here are some hints to create screenshots for your translated user guide.
 Translated images should be placed in a :file:`img/<your_language>/` folder, in
 the same folder as the rst file.
 
-* same environment for all the screen caps (same OS, same decoration, same font
+* Same environment for all the screen caps (same OS, same decoration, same font
   size)
-* use the QGIS -projects included in QGIS-Documentation repository (in
-  :file:`./qgis_projects` ).
+* Use the QGIS -projects included in QGIS-Documentation repository (in
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/writing.rst]: `QGIS projects` instead?

File: source/docs/documentation_guidelines/writing.rst [hunk 4]
```diff
@@ -480,25 +491,25 @@ Here are some hints to create screenshots for your translated user guide.
...
   These were used to produce the 'original' screenshots in the manual.
   The QGIS `Sample Data <https://qgis.org/downloads/data/>`_ (aka Alaska Dataset)
   should be placed in the same folder as the QGIS-Documentation Repository.
-* same size as the english 'original' screenshots, otherwise they will be stretched
+* Same size as the english 'original' screenshots, otherwise they will be stretched
   and look ugly. If you need to have a different size due to longer ui strings,
   don't forget to change the dimension in the rst code of your language.
-* same filename as the english 'original' screenshot.
-* reduce the window to the minimal space needed to show the feature
+* Same filename as the english 'original' screenshot.
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/writing.rst]: Forgot a period? Afaict, not different from lines 494 or 496

File: source/docs/documentation_guidelines/writing.rst [hunk 5]
```diff
@@ -63,6 +63,17 @@ section, subsection and minisec.
    Subminisec
    ^^^^^^^^^^
 
+Lists
+-----
+
+Start all list items with a capital letter.
+
+Do not use punctuation after list items that only contain a single simple
+sentence.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: I agree that much of this is vague.  What I have done is to slightly modify what @SrNetoChan wrote in #3112.
My interpretation of simple is that it does not contain commas, perhaps also not parenthesis.
Is it necessary to be more precise? I think that it is not a problem with different interpretations, but if there are clear rules out there somewhere I am all for including them.

File: source/docs/documentation_guidelines/writing.rst (hunk 1 above)
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
 Use period (``.``) as punctuation for list items that consist of several
```

File: source/docs/documentation_guidelines/writing.rst (hunk 2 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: I am also not sure what is meant, but my interpretation is that a composed sentence would contain commas, semicolons, parenthesis, ...  But if someone can provide clear rules, that would be welcome.

File: source/docs/documentation_guidelines/writing.rst (hunk 3 above)
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
 * Use the QGIS projects included in QGIS-Documentation repository (in
```

File: source/docs/documentation_guidelines/writing.rst [hunk 6]
```diff
@@ -480,25 +491,25 @@ Here are some hints to create screenshots for your translated user guide.
...
   These were used to produce the 'original' screenshots in the manual.
   The QGIS `Sample Data <https://qgis.org/downloads/data/>`_ (aka Alaska Dataset)
   should be placed in the same folder as the QGIS-Documentation Repository.
-* same size as the english 'original' screenshots, otherwise they will be stretched
+* Same size as the english 'original' screenshots, otherwise they will be stretched
   and look ugly. If you need to have a different size due to longer ui strings,
   don't forget to change the dimension in the rst code of your language.
-* same filename as the english 'original' screenshot.
-* reduce the window to the minimal space needed to show the feature
+* Same filename as the english 'original' screenshot.
```
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
 * Same filename as the english 'original' screenshot
```

File: source/docs/documentation_guidelines/writing.rst (hunk 1 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: Done.

File: source/docs/documentation_guidelines/writing.rst (hunk 3 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: Done

File: source/docs/documentation_guidelines/writing.rst (hunk 4 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: Yes! Fixed.

**Comment by havatv** [comment]
Thanks for the feedback, @DelazJ.  As you will see from my comments, I am not too worried about having rules that can be interpreted.  But if someone can find some more precise rules, that would be fine.  I see this as a first attempt.

File: source/docs/documentation_guidelines/writing.rst (hunk 2 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: Rules have now been presented as a list.

**Comment by SrNetoChan** [comment]
About compound sentences: https://examples.yourdictionary.com/compound-sentence-examples.html

File: source/docs/documentation_guidelines/writing.rst [hunk 7]
```diff
@@ -63,6 +63,20 @@ section, subsection and minisec.
...
+Lists are useful for structuring the text.  Here are some simple rules
+common to all lists:
+
+* Start all list items with a capital letter
+
+* Do not use punctuation after list items that only contain a single simple
+  sentence
+
+* Use period ( ``.`` ) as punctuation for list items that consist of several
+  sentences or is composed
```
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
  sentences or are compound
```

File: source/docs/documentation_guidelines/writing.rst [hunk 8]
```diff
@@ -75,7 +75,7 @@ common to all lists:
   sentence
 
 * Use period ( ``.`` ) as punctuation for list items that consist of several
-  sentences or is composed
+  sentences or are compound
```
Comment (SrNetoChan) [inline source/docs/documentation_guidelines/writing.rst]: "several independent sentences or a single compound one"?

File: source/docs/documentation_guidelines/writing.rst [hunk 9]
```diff
@@ -63,6 +63,20 @@ section, subsection and minisec.
...
+Lists are useful for structuring the text.  Here are some simple rules
+common to all lists:
+
+* Start all list items with a capital letter
+
+* Do not use punctuation after list items that only contain a single simple
+  sentence
+
+* Use period ( ``.`` ) as punctuation for list items that consist of several
+  sentences or are compound
```
Comment (havatv) [inline source/docs/documentation_guidelines/writing.rst]:
```suggestion
  sentences or a single compound sentence
```

File: source/docs/documentation_guidelines/writing.rst (hunk 8 above)
Comment (havatv) [reply source/docs/documentation_guidelines/writing.rst]: I hope the change I made is OK.  I don't think the sentences have to be independent.

**Pull request merged**